Formula/cdk.rb

```diff
@@ -1,9 +1,9 @@
 class Cdk < Formula
   desc "Curses development kit provides predefined curses widget for apps"
   homepage "https://invisible-island.net/cdk/"
-  url "https://invisible-mirror.net/archives/cdk/cdk-5.0-20191231.tgz"
-  version "5.0.20191231"
```

Comment:
not quite sure why this did not get caught as audit failure.

While check it later.